### ISSUE TITLE
too many values to unpack error on merge command

### ISSUE THREAD

**Issue by chuckpr** [sanger-pathogens/circlator#44]
I'm getting the following error when running ```circlator merge```:

```
Traceback (most recent call last):
  File "/usr/local/bin/circlator", line 56, in <module>
    exec('circlator.tasks.' + task + '.run()')
  File "<string>", line 1, in <module>
  File "/opt/anaconda/envs/py3.3/lib/python3.3/site-packages/circlator/tasks/merge.py", line 46, in run
    m.run()
  File "/opt/anaconda/envs/py3.3/lib/python3.3/site-packages/circlator/merge.py", line 793, in run
    self.original_fasta, nucmer_coords_file, act_script = self._iterative_bridged_contig_pair_merge(self.outprefix + '.merge')
ValueError: too many values to unpack (expected 3)
```

The commands before ```merge``` appear to be working fine. Not sure what's going on. Any advice?

**Comment by martinghunt** [comment]
What was the command line you ran?

Thanks,
Martin

**Comment by chuckpr** [comment]
Martin,

I ran the following:

```bash
circlator merge --verbose epulo-canu/epulo.consensus.fasta \
test_circ_assemb/scaffolds.fasta \
test_circ_merge
```

-Chuck

**Comment by martinghunt** [comment]
Hi Chuck,

Thanks for reporting that. It's now fixed in the new release 1.1.3.

Martin

**Comment by chuckpr** [comment]
Martin,

Thanks! Much appreciated.

-Chuck
